client/src/styles/CalendarContainer-style.jsx

```diff
@@ -1,10 +1,12 @@
 import styled from 'styled-components';
 
 const Wrapper = styled.div`
-  background: aliceBlue;
-  display: inline-block;
+  height: 90px;
+  padding: 3px;
+  display: flex;
   white-space: nowrap;
-  overflow-x: hidden;
+  overflow-x: scroll;
+  scroll-snap-type: x proximity;
 `;
```

Comment:
Not sure if you have the scrollbar disappearing with a container but this will also do it &::-webkit-scrollbar {
  display: none;
}

Reply:
Thanks Rob!